src/frontend/sharedComponents/SelectOne.tsx

```diff
@@ -26,16 +31,35 @@ export const SelectOne = <T,>({
             : index
         }
         testID={`${item.value}LanguageButton`}
-        onPress={() => value !== item.value && onChange(item.value)}>
+        onPress={() => value !== item.value && onChange(item.value)}
+        style={[radioButtonPosition === 'right' && styles.rowReverse]}>
         <ListItemIcon
           iconName={
             item.value === value
               ? 'radio-button-checked'
               : 'radio-button-unchecked'
           }
+          style={[
+            radioButtonPosition === 'right'
+              ? styles.alignRight
+              : styles.alignLeft,
+          ]}
```

Comment:
nit: no need for this to be within an array

```suggestion
          style={
            radioButtonPosition === 'right'
              ? styles.alignRight
              : styles.alignLeft,
          }
```